q2_types/feature_table/_transformer.py

```diff
@@ -12,26 +12,36 @@
 import pandas as pd
 import qiime
 
-from . import BIOMV1Format
+from . import BIOMV1Format, BIOMV210Format
 from ..plugin_setup import plugin
 
 
+def _get_generated_by():
+    return 'qiime %s' % qiime.__version__
+
+
 @plugin.register_transformer
 def _1(data: biom.Table) -> BIOMV1Format:
+    # TODO: issue a warning about limited support for this format
```

Comment:
I'd argue it's a position that qiime2 can take. BIOM just defines the format and a Python-based in memory representation. The benefit of taking a position in qiime2 is that we encourage developers of plugins to get up to speed, which opens the door to random access without having to worry about transformation overhead.